examples/interoperability/app.js

```diff
@@ -0,0 +1,125 @@
+import 'babel-polyfill';
+/* global document, window,*/
+/* eslint-disable no-console */
+import React, {PureComponent} from 'react';
+import {render} from 'react-dom';
+import DeckGL, {
+  ScatterplotLayer, OrthographicViewport, COORDINATE_SYSTEM
+} from '../../src';
```

Comment:
> Once those changes are landed, I will change back import from '../../src' to 'deck.gl'.

If you look at the other examples, they allow you to choose whether to run from the installed version or from the local source `npm start` vs `npm run start-local`. My point is that by following the same setup, you can have both models working, and you can debug from src now without npm link etc (the `webpack.config.local` in `/examples` parent dir takes care of aliasing all the modules so that npm link is not needed).